wasm/jit/compiler.go

```diff
@@ -8,7 +8,7 @@ type compiler interface {
 	// emitPreamble is called before compiling any wazeroir operation.
 	// This is used, for example, to initilize the reserved registers, etc.
 	emitPreamble()
-	// Finilizes the compilation, and returns the byte slice of native codes.
+	// Finalizes the compilation, and returns the byte slice of native codes.
```

Comment:
should we rename to "generate"? Ex this is the final step where the final code is generated (if code is generated from intermediate steps is incidental as this is the only place you can get it from the api)

Reply:
sg